src/components/CropSelector/CropTable.js

```diff
@@ -575,23 +580,26 @@ const CropTableComponent = (props) => {
                   borderRight: '5px solid white',
                 }}
               >
-                  <Button onClick={sortCropsByName}>
-                    {nameSortFlag ? (
-                      <Sort
-                        style={{
-                          color: CustomStyles().secondaryProgressBtnColor,
-                        }}
-                      />
-                    ) : (
-                      <Sort
-                        style={{
-                          color: CustomStyles().progressColor,
-                          transform: 'rotate(180deg)',
-                        }}
-                      />
-                    )}
-                    &nbsp; <Typography variant="body1" style={{ color: '#000' }}>COVER CROPS</Typography>
-                  </Button>
+                <Button onClick={sortCropsByName}>
+                  {nameSortFlag ? (
+                    <Sort
```

Comment:
similar to below.  this can be simplified.... and we are seeing this and the one below are the exact same code.  We should be able to reuse this code.  feel free to try to tackle this on your own but we can plan to work on this together Thursday if you'd like.